### ISSUE TITLE
Nautobot container scores an "F" from security scanner

### ISSUE THREAD

**Issue by nniehoff** [nautobot/nautobot#1111]
### Environment
* Python version:  3.6
* Nautobot version:  1.1.5

Artifacthub scans the docker containers in use in a helm chart to provide users with a security report of the containers they are deploying.  Nautobot currently scores an "F" with over 100 issues.  Artifacthub uses a tool called [trivy](https://github.com/aquasecurity/trivy) from Aquasec which is a fairly large name in the container security space.

See [the report](https://artifacthub.io/packages/helm/nautobot/nautobot?modal=security-report)

**Comment by bryanculver** [comment]
This is a byproduct of pulling from the python-slim latest image. Because it is based off of the Debian 11.2 image it is reporting vulnerabilities that may be resolved in our apt updates layer.

Unless we can force-target a newer version of Debian we likely won't produce a higher score.

We can revisit this if we start running into deployment roadblocks.

**Comment by rifen** [comment]
Nautobot Version: 1.2.3
Container Scanner: twistlock
Today my org began questioning me because of this poor score :) I got a related list to nniehoff but seems more "updated":
Critical (3)
[CVE-2022-22822](https://security-tracker.debian.org/tracker/CVE-2022-22822) | [CVE-2022-22823](https://security-tracker.debian.org/tracker/CVE-2022-22823) | [CVE-2022-22824](https://security-tracker.debian.org/tracker/CVE-2022-22824)
High (6)
CVE-2021-46143 | CVE-2022-22827 | CVE-2022-22825 | CVE-2022-22826 | PRISMA-2021-0041 | CVE-2021-23727

The ones listed mostly have to do with a libexpat the most it seems. Idk if it can just be removed to reduce this noise or what options I would have from my side. I am using a custom image with a Dockerfile so that can be done from my side but that would probably affect me in future versions.
Affected: https://github.com/libexpat/libexpat
Fix: https://github.com/libexpat/libexpat/pull/543

**Comment by bryanculver** [comment]
Thanks for the feedback @rifen. Unfortunately as these dependencies are being brought in via the Debian base image our base image, python-slim, is based off of, it's a challenge to identify what files would need to be removed but what impact it's removal would cause and if the alerts would still be presented if we added the removals as an additional layer.

For what it's worth even projects like [Jenkins](https://artifacthub.io/packages/helm/jenkinsci/jenkins) and [Redis](https://artifacthub.io/packages/helm/bitnami/redis) are just as affected by vulnerable base images with no fixes.

However now that we are aware of organization having alerts tuned to catch this issue (and recent CVEs in direct dependencies that we have addressed recently: #1238 & #1267) we are starting discussions on how we can minimize vulnerabilities from dependencies of our dependencies.
